bananas_api/helpers/api_schema.py

```diff
@@ -77,12 +82,12 @@ class Global(OrderedSchema):
     read_only = ["archived", "replaced_by"]
 
     # Most of these limits are limitations in the OpenTTD client.
-    name = fields.String(validate=validate.Length(max=31))
+    name = fields.String(validate=validate.BytesLength(max=31))
```

Comment:
I am 100% sure you did not test this. don't be lazy, start a local setup, and make sure this works please. README is hopefully pretty clear, and via docker it is really easy. Please don't be "that guy" :D :D